app/server/utils/user.py

```diff
@@ -504,6 +504,9 @@ def proccess_create_or_modify_user_request(attribute_dict,
 
         return response_object, 200
 
+    #TODO(admin_create): add enrolled by if passed in
+    #TODO(admin_create): add bio, gender
+    #TODO(admin_create): if passed in default token, use that token for default transfer account
```

Comment:
is it possible to set the token on the transfer account created if it's being done by the eth worker?

Reply:
what do u mean by "being done by the eth worker"? In general eth worker is pretty passive... it takes blockchain txn requests from the main app and tries to do them.

Reply:
ohh okay, i haven't dug into that code, but this means transfer account is created on the app side and not on eth worker? is it created after eth worker finishes running then? it sets the blockchain address on the transfer account based on what's returned from the eth worker right?

Reply:
okay i see the code where transfer account is created as part of the user creation flow, but i don't see where it calls the eth worker. if we provide a public serial number instead of a blockchain address, it just creates a transfer account without an address...? where would it update that transfer account to the newly created wallet address?

Reply:
oh wait i see it. on transfer account init, if no blockchain address is created it kicks off a celery worker that sync will return the blockchain address from wallet creation. is that correct?

Reply:
^^ yep that's correct. Creating a blockchain wallet is (effectively) instant on the eth worker's side, which is why the transfer_account waits for the result

Reply:
so to draw a conclusion under the whole thing, yes `transfer_account.token = token` is a valid operation